docs/src/tutorials/resolves.md

```diff
@@ -0,0 +1,627 @@
+```@meta
+DocTestFilters = [r"≤|<=", r" == | = ", r" ∈ | in ", r" for all | ∀ ", r"d|∂", 
+                  r"E|𝔼", r"integral|∫"]
+```
+
+# Re-solve Guide
+There are situations where one may want to re-solve problems defined in `InfiniteOpt`, such as in model predictive control. InfiniteOpt's re-solve framework facilitates this through a persistent solver interface and model backend, avoiding the need to build them again for each re-solve.
+
+Below, we show how to do efficient re-solves via model predictive control of a continuous stirred tank reactor (CSTR). Please refer to the 
+Guide for more complete information on modeling a stand-alone problem.
+
+## Preliminaries 
+### Software Setup
+First, we need to make sure everything is installed. This will include:
+- installing Julia 
+- installing the `InfiniteOpt.jl`, `JuMP.jl`, and `DifferentialEquations.jl` packages
```

Comment:
I don't think JuMP is needed (it installs with InfiniteOpt) and you should indicate why DifferentialEquations is needed.